components/authz-service/server/v2/project.go

```diff
@@ -155,23 +155,37 @@ func (s *state) UpdateProject(ctx context.Context,
 			"error converting project with ID %q: %s", resp.ID, err.Error())
 	}
 
-	err = s.projectUpdateManager.Start()
+	return &api.UpdateProjectResp{Project: apiProject}, nil
+}
+
+func (s *state) ApplyRulesStart(ctx context.Context, req *api.ApplyRulesStartReq) (*api.ApplyRulesStartResp, error) {
+	log.Info("apply project rules: START")
```

Comment:
❓ `log`? Looks like this was **both** using the [global logrus logger](https://github.com/chef/automate/blob/fb00988690ad8fa4c56d661abb2f20a35c4abd4f/components/authz-service/server/v2/project.go#L18) and having [a logger in its state](https://github.com/chef/automate/blob/fb00988690ad8fa4c56d661abb2f20a35c4abd4f/components/authz-service/server/v2/project.go#L82). Let's avoid that, please 😉 
```suggestion
	s.log.Info("apply project rules: START")
```
same goes for the other `log.Info` calls below.

On that note, if we want every method to spit its name into our logs when called, we should do this via a grpc middleware. The gateway has a useful example wired up, [see here](https://github.com/chef/automate/blob/c32c9b7f5bdcd923008504f36a75456786ee9b0f/components/automate-gateway/gateway/server.go#L265-L287).